rj_gameplay/rj_gameplay/tactic/nmark_tactic.py

```diff
@@ -119,7 +119,7 @@ def get_requests(
         return role_requests
 
     def tick(self, world_state: rc.WorldState,
-             role_results: tactic.RoleResults) -> List[tactic.SkillEntry]:
+             role_results: tactic.RoleResults, props: Optional[tactic.PropT]) -> List[tactic.SkillEntry]:
```

Comment:
Error: rj_gameplay/rj_gameplay/tactic/nmark_tactic.py:121: error: Signature of "tick" incompatible with supertype "ITactic"
-> If the method given (tick) was meant to override or overload the parent then it doesn't do that.  The solution was to just add the props as an optional parameter.
-> This shouldn't break anything, but if it does then I'll fix things.